suite-native/atoms/src/Sheet/BottomSheetFlashList.tsx

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useRef, ReactNode } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import Animated from 'react-native-reanimated';
+import { PanGestureHandler } from 'react-native-gesture-handler';
+import { GestureResponderEvent, Pressable } from 'react-native';
+
+import { FlashList, FlashListProps } from '@shopify/flash-list';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from '../Box';
+import { BottomSheetContainer } from './BottomSheetContainer';
+import { useBottomSheetAnimation } from './useBottomSheetAnimation';
+import { BottomSheetHeader } from './BottomSheetHeader';
+
+export type BottomSheetFlashListProps<TItem> = {
+    isVisible: boolean;
+    isCloseDisplayed?: boolean;
+    onClose: (isVisible: boolean) => void;
+    title?: ReactNode;
+    subtitle?: ReactNode;
+    ExtraProvider?: React.ComponentType;
+    estimatedListHeight?: number;
```

Comment:
Would it make sense to swap these two lines?
```suggestion
    estimatedListHeight?: number;
    ExtraProvider?: React.ComponentType;
```

Reply:
I don't see any benefit doing that manually, if you like it in some specific order there are ESLint plugins like this one https://github.com/azat-io/eslint-plugin-perfectionist where you can setup everything in this matter. I think it would be nice to have something like this implemented, feel free to prepare PR if you are unhappy with current state.

Reply:
I don't think such a plugin would make it better. From my point of view, semantic sorting is the best approach and I don't think it can be done automagically. At least not yet.

Anyway, I'm not unhappy, I'm just asking what you think.